.drone.yml

```diff
@@ -79,6 +88,17 @@ pipeline:
       event: push
       branch: [ master, release-* ]
 
+  scan-platform:
+    image: astronomerio/ap-build:0.1.1
+    commands:
+      - make scan-platform
+    environment:
+      - ASTRONOMER_VERSION=0.10.0
```

Comment:
What's this used for?

Reply:
Need remove it

Reply:
In the `build-platform` stage we have this as `0.0.0` in the this file and it is over-written by the `bin/release-platform` script. We should probably the same here.